src/app/client/src/app/modules/groups/components/add-member/add-member.component.spec.ts

```diff
@@ -111,10 +111,10 @@ describe('AddMemberComponent', () => {
     description: '',
     membershipType: GroupMembershipType.INVITE_ONLY,
     active: true,
-    isActive() { return true ;}
+    isActive() { return true ; }
   };
 
-    component.memberId = '2';
+    component.memberId = 'test-1234     abc';
```

Comment:
no need to re-assign **component.memberId** inside each it() block, since u r doing already in beforeEach(), so it will be assigned every time.

Reply:
ok

Reply:
if we didn't re-assign then it takes the value as 2 .and gives the test cases fail

Reply:
it should not fail. check again.